environments/chaotic_pendulum.py

```diff
@@ -58,6 +58,12 @@ def get_max_noise_std(self):
         """Return maximum noise std that keeps the environment stable."""
         return 0.05
 
+    def get_default_radius_bounds(self):
+        """Returns:
+            radius_bounds (tuple): (min, max) radius bounds for the environment.
+        """
+        return (1.3, 2.3)
```

Comment:
for chaotic pendulum i use (0.5, 1.3). I think it generates diverse data and (1.3, 2.3) might contain too much momentum. 
```suggestion
        return (0.5, 1.3)
```